carma-messenger-ui/website/scripts/main.js

```diff
@@ -0,0 +1,109 @@
+
+//Load global session variables
+var EventSessionFormFields = LoadSessionVariables();
+
+$(document).ready(function () {
+    $('#card-truck-inspection').click(function(){
+       $('#divCarmaMessengerView').css('display','');
+       $('#divWidgetArea').css('display','');
+       $('#divWidgetAreaEventManagement').css('display','none');
+       $('#Messenger_back_arrow').css('display','inline-block');
+       $('#divCarmaMessengerMenu').css('display','none');
+
+       //show truck inspection widget
+       CarmaJS.WidgetFramework.closeWidgets();
```

Comment:
As mentioned on our meeting, please make sure to record that this Truck Inspection functionality is still working for sprint review to ensure regression testing is done.

Reply:
Just a suggestion on next updates you make to add more widgets, maybe update the close and load function names to say "closeTruckInspectionWidgets" and "loadTruckInspectionWidgets" to match the naming convention you have below.